db/seeds.js

```diff
@@ -41,8 +56,10 @@ const reset = async () => {
       ]
     }
   ])
+  return (userCreate, bookCreate)
 }
 
+//why is response only pulling the number from the 1st databse
```

Comment:
Not sure what you mean by this, but you can't return two things. You could return something like the following instead:
```js
[ userCreate, bookCreate ]
```